Add info on n8n-node CLI tool (#3505)
n8n-io/n8n-docs: docs/integrations/creating-nodes/build/reference/verification-guidelines.md

Consider following these guidelines while building your node if you want to submit it for verification by n8n. Any user with verified community nodes enabled can discover and install verified nodes from n8n's nodes panel across all deployment types (self-hosted and n8n Cloud).
///

> ## Must use the n8n-node tool

Reviewer: Maybe it's fine, but "must" here and below sounds very strict to me

> ## Must use the n8n-node tool

Author: ```suggestion
## Use the n8n-node tool
```

> All verified community nodes must use the [`n8n-node` tool](/integrations/creating-nodes/build/n8n-node.md) to create and check their package. This helps n8n ensure quality and consistency by:

Author: ```suggestion
All verified community nodes should strongly consider using the [`n8n-node` tool](/integrations/creating-nodes/build/n8n-node.md) to create and check the package. This helps n8n ensure quality and consistency by:
```
